calculators/gas_king_rating

```diff
@@ -0,0 +1,101 @@
+<?php
+
+$GLOBALS['rating_calculators']['gas_king_rating'] = [
+    'description' => '
+        gas king rating
+        only objective: gas as much and as soon as you can
+
+        you start with 2000 rating (compression is activated although maybe not efficient, I was too lazy to see how you did it)
+        you lose 10 rating per minute for each gas not taken
+        you win 1, 2, 3, 4 rating per minute for gas 1, 2, 3, 4
+        ',
+    'default_rating' => 2000,
+    'fields' => [
+        'g.duration',
+        'gp.gas_timings',
+        'gp.gas_count',
+        ],
+    'main' => 'dsr_gas_king_rating__main',
+    ];
+
+function dsr_gas_king_rating__same_sign($a, $b)
```

Comment:
right after the meta info, first function should be the main one. For the readability purposes. Only later, where noone reads, you declare helper functions. Recommendations are to place functions in order they are getting called. This is no C, functions work no matter where you've declared it.

Reply:
Ok

Reply:
I moved the sign function at the end of the code, fixed